cmd/kubelet/app/server.go

```diff
@@ -256,8 +262,10 @@ is checked every 20 seconds (also configurable with a flag).`,
 				return fmt.Errorf("failed to validate kubelet configuration, error: %w, path: %s", err, kubeletConfig)
 			}
 
+			// Retrieve the logger again to reflect the updated log settings
+			logger = klog.FromContext(ctx)
```

Comment:
please move this to the line 253 right after the `logs.InitLogs()`

Reply:
@SergeyKanzhelev 
Thank you.
I fixed here 👍https://github.com/kubernetes/kubernetes/compare/99a9ed01d8b150df297c5558c403d85a5fdcdc90..2982f299c340da68bfd90dadd926e792ccc56c51